### ISSUE TITLE
illegal character U+0040 (SyntaxError)

### ISSUE THREAD

**Issue by Ildar1313** [dynamia-projects/chartjs-zk#13]
I use the ZK framework.
I have a ChartjsData:

` public ChartjsData getChartData() {
        CategoryChartjsData chartData = new CategoryChartjsData();
        chartData.add("In", new Random().nextInt(10000));
        chartData.add("Out",  new Random().nextInt(10000));
        return chartData;
    }`

When I click on the button, the chart should be updated:

   `@Command
    @NotifyChange({"chartData"})
    public void updateChart(){
    }`

 But an error occurs: "(illegal character U+0040 (SyntaxError))"

**Comment by Ildar1313** [comment]
Do you have any plans to update the chart?

I am connecting components in a ViewModel.
I create a chart component and put the component in a window. And each time to update the chart, I delete the chart component, create a new one and redraw the window:

`@Wire("#centerChart")
    private Center center;

@AfterCompose
    public void afterCompose(@ContextParam(ContextType.VIEW) Component window) {
        this.window = window;
        Selectors.wireComponents(window, this, false);
    }

@Command
    public void updateChart() {
        if (this.chart != null) {
            chart.detach();
            window.invalidate(); 
        }
        chart = Executions.createComponents("charts.zul", window, args);
        center.appendChild(chart);
    }
`
